scripts/e2e

```diff
@@ -25,7 +25,9 @@ npx hardhat migrate:unpause --network localhost --graph-config config/graph.loca
 ### Test
 # Run tests using the localhost evm instance and the localhost graph config
 npx hardhat e2e --network localhost --graph-config config/graph.localhost.yml
-
+npx hardhat e2e:scenario create-subgraphs --network localhost --graph-config config/graph.localhost.yml
```

Comment:
```
  if (chainId == 1337) {
    await (cli.wallet.provider as providers.JsonRpcProvider).send('evm_setAutomine', [false])
  }
```
That line after `migrate` finishes is making the rest of the tests to run slow. What if we set automine:true by default for local in the hardhat.config?

This is the current config:
```
      mining: {
        auto: false,
        interval: 13000,
      },
```

And then we set to false for specific tests. Seems to me that most of the time we want it to be fast during tests.

Reply:
I think I agree with this. With the intent of being less disruptive for now I've added a parameter to the `migrate` call so we have the flexibility to leave auto mine on or off after the deployment is done (and assuming we are in 1337 ofc).